src/scss/style.scss

```diff
@@ -0,0 +1,92 @@
+.module {
+    color: #625F5F;
+    border-radius: 2px;
+    padding: 15px;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #C8C4C3;
+    min-height: 100vh;
+
+    .wrapper {
+        display: flex;
+        border: 1px solid #C8C4C3;
+        border-radius: 10px;
+        justify-content: center;
+        flex-direction: column;
+        -webkit-box-shadow: 6px 9px 5px -4px rgba(0,0,0,0.44);
```

Comment:
You are using sass, this can be extracted into a mixin or a function. It's not a big deal but still, it's a way to clear the code.